src/dialog-controller.js

```diff
@@ -76,11 +76,15 @@ export class DialogController {
             } else {
               this._reject(new DialogCancelError(output));
             }
-            return result;
+            return { wasCancelled: false };
```

Comment:
Should we be returning a result type here?  Probably doesn't matter much either way just curious (we may have talked about this before)